vllm/lora/punica_wrapper/punica_selector.py

```diff
@@ -1,19 +1,13 @@
 from vllm.platforms import current_platform
-from vllm.utils import print_info_once
+from vllm.utils import print_info_once, resolve_obj_by_qualname
 
 from .punica_base import PunicaWrapperBase
 
 
 def get_punica_wrapper(*args, **kwargs) -> PunicaWrapperBase:
-    if current_platform.is_cuda_alike():
-        # Lazy import to avoid ImportError
-        from vllm.lora.punica_wrapper.punica_gpu import PunicaWrapperGPU
-        print_info_once("Using PunicaWrapperGPU.")
-        return PunicaWrapperGPU(*args, **kwargs)
-    elif current_platform.is_hpu():
-        # Lazy import to avoid ImportError
-        from vllm.lora.punica_wrapper.punica_hpu import PunicaWrapperHPU
-        print_info_once("Using PunicaWrapperHPU.")
-        return PunicaWrapperHPU(*args, **kwargs)
-    else:
-        raise NotImplementedError
+    punica_wrapper_qualname = current_platform.get_punica_wrapper()
+    punica_wrapper_cls = resolve_obj_by_qualname(punica_wrapper_qualname)
+    punica_wrapper = punica_wrapper_cls(*args, **kwargs)
+    assert punica_wrapper is not None
```

Comment:
I think this assertion will never work, because `NotImplementedError` will be raised when getting punica warper.

Reply:
But when the returned `punica_wrapper_qualname` was wrong, e.g. misspelling, this assertion could be useful.

Reply:
Okay, I got your point. Then could you add a assertion info here to instruct users to check the correctness of `punica_warpper_qualname`?

Reply:
Done.